javascript/javascript2/week2/currencyCnvrtrWeek2.js

```diff
@@ -0,0 +1,132 @@
+// currencyConverter.js
+
+const exRates = {
+  timestamp: Date.now(),
+  base: "USD",
+  date: new Date().toISOString().split("T")[0],
+  exRates: {
+    USD: 1.23396,
```

Comment:
That is some funny maths: `1 USD = 1.23396 USD`

Reply:
Hi vdaguenet, what should I need to do with the above code? Need to change the value or ? Otherwise base is USD, so 1 USD is = to 1.23396 USD